docker-compose-cdot-deployment.yml

```diff
@@ -0,0 +1,213 @@
+      CONFIG_TMDD_VALIDATION_PERIOD_MINUTES: ${TASKS_CONFIG_TMDD_VALIDATION_PERIOD_MINUTES}
+      CONFIG_RETENTION_ENFORCEMENT_PERIOD_MINUTES: ${TASKS_CONFIG_RETENTION_ENFORCEMENT_PERIOD_MINUTES}
+      CONFIG_RETENTION_REMOVE_TIMS: ${TASKS_CONFIG_RETENTION_REMOVE_TIMS}
+      CONFIG_RETENTION_REMOVE_STATUS_LOGS: ${TASKS_CONFIG_RETENTION_REMOVE_STATUS_LOGS}
+      CONFIG_HSM_FUNCTIONALITY_MINUTES: ${TASKS_CONFIG_HSM_FUNCTIONALITY_MINUTES}
+      CONFIG_HSM_URL: ${TASKS_CONFIG_HSM_URL}
+      CONFIG_HSM_ERROR_EMAIL_FREQUENCY_MINUTES: ${TASKS_CONFIG_HSM_ERROR_EMAIL_FREQUENCY_MINUTES}
+      CONFIG_RUN_HSM_CHECK: ${TASKS_CONFIG_RUN_HSM_CHECK}
+      CONFIG_SDW_TTL: ${TASKS_CONFIG_SDW_TTL}
+      CONFIG_DEFAULT_LANE_WIDTH: ${TASKS_CONFIG_DEFAULT_LANE_WIDTH}
+      CONFIG_RSU_ROUTES: ${TASKS_CONFIG_RSU_ROUTES}
+      CONFIG_POINT_INCIDENT_BUFFER_MILES: ${TASKS_CONFIG_POINT_INCIDENT_BUFFER_MILES}
+    logging:
+      options:
+        max-size: "20m"
+        max-file: "3"
+    depends_on:
+      - ode_wrapper
+      - cv-data-controller
+      - rsu-data-controller
+
+  tim-refresh:
+    build: tim-refresh
+    restart: always
+    environment:
+      CONFIG_ODE_URL: ${REFRESH_CONFIG_ODE_URL}
+      CONFIG_SDW_TTL: ${REFRESH_CONFIG_SDW_TTL}
+      CONFIG_CV_REST_SERVICE: ${REFRESH_CONFIG_CV_REST_SERVICE}
+      CONFIG_SDW_REST_URL: ${SDW_REST_URL}
+      CONFIG_SDW_API_KEY: ${SDW_API_KEY}
+      CONFIG_DEFAULT_LANE_WIDTH: ${REFRESH_CONFIG_DEFAULT_LANE_WIDTH}
+      CONFIG_RSU_ROUTES: ${REFRESH_CONFIG_RSU_ROUTES}
+      CONFIG_POINT_INCIDENT_BUFFER_MILES: ${REFRESH_CONFIG_POINT_INCIDENT_BUFFER_MILES}
+      CRON_EXPRESSION: ${REFRESH_CRON_EXPRESSION}
+      CONFIG_ALERT_ADDRESSES: ${REFRESH_CONFIG_ALERT_ADDRESSES}
+      CONFIG_FROM_EMAIL: ${REFRESH_CONFIG_FROM_EMAIL}
+      CONFIG_ENVIRONMENT_NAME: ${ENVIRONMENT_NAME}
+      CONFIG_MAIL_HOST: ${MAIL_HOST}
+      CONFIG_MAIL_PORT: ${MAIL_PORT}
+    logging:
+      options:
+        max-size: "20m"
+        max-file: "3"
+
+  cv-data-controller:
+    build: cv-data-controller
+    restart: always
+    ports:
+      - "8888:8888"
+    environment:
+      SERVER_PORT: ${CONTROLLER_SERVER_PORT}
+      CONFIG_DB_URL: ${CONTROLLER_CONFIG_DB_URL}
+      CONFIG_DB_USERNAME: ${CONTROLLER_CONFIG_DB_USERNAME}
+      CONFIG_DB_PASSWORD: ${CONTROLLER_CONFIG_DB_PASSWORD}
+      CONFIG_MAXIMUM_POOL_SIZE: ${CONTROLLER_CONFIG_MAXIMUM_POOL_SIZE}
+      CONFIG_CONNECTION_TIMEOUT: ${CONTROLLER_CONFIG_CONNECTION_TIMEOUT}
+      CONFIG_ODE_URL: ${CONTROLLER_CONFIG_ODE_URL}
+      SPRING_DATA_NEO4J_URI: ${CONTROLLER_SPRING_DATA_NEO4J_URI}
+      CONFIG_ALERT_ADDRESSES: ${CONTROLLER_CONFIG_ALERT_ADDRESSES}
+      CONFIG_FROM_EMAIL: ${CONTROLLER_CONFIG_FROM_EMAIL}
+      CONFIG_ENVIRONMENT_NAME: ${ENVIRONMENT_NAME}
+      CONFIG_MAIL_HOST: ${MAIL_HOST}
+      CONFIG_MAIL_PORT: ${MAIL_PORT}
+      LOGGING_LEVEL_ROOT: ${CONTROLLER_LOGGING_LEVEL_ROOT}
+    logging:
+      options:
+        max-size: "20m"
+        max-file: "3"
```

Comment:
Question (non-blocking): Should confluent properties be included in any of these services?

Reply:
I've double-checked and everywhere that confluent properties are required they are present